File: Excel_Programs/C++/question2.cpp
// max and min element in array
#include<iostream>
using namespace std;

int main()
{
    int n,max,min;
    cout<<"Enter the number of elements to be entered : ";
    cin>>n;
    int arr[n];
    for(int i =0;i<n;i++)
    {
        cout<<"Enter a number : ";
        cin>>arr[i];
    }
    min=arr[0],max=0;
    for (int i =0; i<n;i++)
    {
        if(max<arr[i])
        {
            max= arr[i];
        }
        if(min>arr[i])
        {
            min=arr[i];
        }
    }
    cout<<"The max number in the array : "<<max<<"\n";
    cout<<"The min number in the array : "<<min<<"\n";
}

File: Excel_Programs/Java/question2.java
// max and min in array 
import java.util.*;
class question2 
{
    public static void main(String[] args) 
    {
        Scanner sc = new Scanner(System.in);
        int max=0,min,n;
        System.out.print("Enter the number of elements to be entered : ");
        n = sc.nextInt();
        int arr[] = new int[n];
        for (int i =0 ; i<n;i++)
        {
            System.out.print("Enter a number : ");
            arr[i]=sc.nextInt();
        }
        System.out.println("The number entered in the array : ");
        for(int i =0 ; i<n ; i++)
        {
            System.out.print(arr[i]+" ");
        }
        min =arr[0];
        System.out.println();
        for(int i =0;i<n;i++)
        {
            if(arr[i]>max)
            {
                max=arr[i];
            }
            if(min>arr[i])
            {
                min=arr[i];
            }
        }
        System.out.println("The max element in the array : "+max);
        System.out.println("The min element in the array : "+min);
        sc.close();
    }
}
